src/Uno.UWP/Globalization/NumberFormatting/NumeralSystemTranslator.cs

```diff
@@ -0,0 +1,167 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Windows.Globalization.NumberFormatting
+{
+	public partial class NumeralSystemTranslator
+	{
+		private static Dictionary<string, char[]> s_numeralSystemToDigitsDictionary =
+			NumeralSystemTranslatorHelper.CreateNumeralSystemToDigitsDictionary();
+		private string numeralSystem;
+
+		public string NumeralSystem
+		{
+			get => numeralSystem;
+			set
+			{
+				if (!s_numeralSystemToDigitsDictionary.ContainsKey(value))
+				{
+					throw new ArgumentException("The parameter is incorrect.");
+				}
+
+				numeralSystem = value;
+			}
+		}
+
+		public IReadOnlyList<string> Languages { get; }
+
+		public string ResolvedLanguage { get; }
```

Comment:
Please make the ordering in the class as follows:

- Fields
- Constructors
- Events
- Properties
- Methods